docker/analysis-tools/Dockerfile

```diff
@@ -85,3 +85,13 @@ RUN curl -sSL https://sdk.cloud.google.com > /tmp/gcl && bash /tmp/gcl --install
 # Configure gcloud
 ENV PATH $PATH:/usr/gcloud/google-cloud-sdk/bin
 RUN gcloud auth activate-service-account --key-file=$GCS_AUTH_FILE
+
+# Add Cromwell Credentials
+ADD broad-dsde-mint-dev-cromwell.json /usr/secrets/broad-dsde-mint-dev-cromwell.json
```

Comment:
I thought you were just updating to switch to 16.04 -- why is this being added here? Similarly for sratoolkit below.

Reply:
Actually, the analysis-tools in master branch is outdated, @jishuxu updated this docker in her branch, and I just want to include those changes here together with the Ubuntu version update.

Reply:
it is analysis docker, not stored in quay. I added more packages as I needed to run analysis.